examples/spl_token.sol

```diff
@@ -0,0 +1,135 @@
+import 'solana';
```

Comment:
This file should also be on codegen/contract tests, as it should never be broken.

Reply:
We have tests for that.